testbed/testbed/test_case.go

```diff
@@ -226,9 +219,8 @@ func (tc *TestCase) StopLoad() {
 
 // StartBackend starts the specified backend type.
 func (tc *TestCase) StartBackend() {
-	if err := tc.MockBackend.Start(); err != nil {
-		tc.t.Fatalf("Cannot start backend: %s", err.Error())
-	}
+	err := tc.MockBackend.Start()
+	require.NoError(tc.t, err, "Cannot start backend")
```

Comment:
This can be inlined. Same for everywhere :)

Reply:
Everywhere?  I can see how it can be done for things that only return err, but not for places where a result is also return, like at line 110 where it calls filepath.Abs.

Reply:
Everywhere where is possible :)